packages/test/test-end-to-end-tests/src/test/gc/gcVersionUpgrade.spec.ts

```diff
@@ -36,7 +36,7 @@ class ContainerRuntimeFactoryWithGC extends ContainerRuntimeFactoryWithDefaultDa
     public async instantiateRuntime(
         context: IContainerContext,
     ): Promise<IRuntime> {
-        const runtime = await super.instantiateRuntime(context);
+        const runtime = await super.instantiateRuntime(context, true);
```

Comment:
Not sure how this is going to work out... glancing through the test provider code, this will be used by the test loader, which in turn could be used to either create new or load existing.

@andre4i - Am I right to be concerned?  In the new way of managing `existing` flag, how would that bit be plumbed through?

Reply:
This likely needs to have a similar logic like the `RuntimeFactoryHelper`